.github/CODEOWNERS

```diff
@@ -1,8 +1,28 @@
 # See for instructions on this file https://help.github.com/articles/about-codeowners/
 
-# Instructions for subscribing to scheduled build failure notifications:
-# https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/codeowners-notifications.md
+# Instructions for CODEOWNERS file format and automatic build failure notifications:
+# https://github.com/Azure/azure-sdk/blob/master/docs/engineering-system/codeowners.md
 
+###########
+# Eng Sys
+###########
+/eng/   @weshaggard @scbedd
+
+/eng/pipelines/ @weshaggard @scbedd @mitchdenny @danieljurek
```

Comment:
I don't think we need to split these out. We shoudl just have `/eng/` and the yml file wildcards.